Evaluation/Eval_5Dec20/E_05Dec_hruday/Eval_hruday_loginpage.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login Page</title>
+</head>
+<link rel="stylesheet" href="Eval_hruday_5th_Dec_style.css">
+<body>
+    <h1 class="color1" style="text-align: center;"> <b> Create Your Profile </b></h1>
+
+
+        
+      
+        <div class="logindiv">
+          <label for="uname" class="m-l"><b>Username:</b></label> <br>
+          <input type="text" placeholder="Enter Username" name="uname" class="u_p" required>
+      <br>
+          <label for="psw" class="m-l"><b>Password:</b></label> <br>
+          <input type="password" placeholder="Enter Password" name="psw" class="u_p" required>
+          <br>
+          <label>
+            <input type="checkbox" checked="checked" name="remember" style="margin-top: 5px;"> Remember me
+          </label> 
+          <br>
+          <br>
+
+          <div class="center" >
+            <a href="Eval_hruday_userdetails.html" target="_blank"><button class="button">Signup</button></a> 
+            </div>
+          
+        </div>
+
```

Comment:
remove extra lines in this code and in others too

Reply:
okkk..